stories/SQFormScrollableCardsMenuWrapper.stories.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import * as yup from 'yup';
+
+import {
+  SQFormScrollableCardsMenuWrapper,
+  SQFormScrollableCard,
+  SQFormTextField,
+  SQFormCheckbox
+} from '../src';
+import {createDocsPage} from './utils/createDocsPage';
+
+export default {
+  title: 'Forms/SQFormScrollableCardsMenuWrapper',
+  component: SQFormScrollableCardsMenuWrapper,
+  parameters: {
+    docs: {page: createDocsPage({showStories: false})}
+  }
+};
+
+export const sqFormScrollableCardsMenuWrapper = () => {
+  const menuItems = [
+    {
+      label: 'Details',
+      value: 'details'
+    },
+    {
+      label: 'Permissions',
+      value: 'permissions'
+    }
+  ];
+
+  return (
+    <div style={{width: '100%', height: '100%'}}>
+      <SQFormScrollableCardsMenuWrapper
+        title="[selected user]"
+        menuItems={menuItems}
```

Comment:
With the recent changes, we can remove this prop from the story
We can also remove the comments below mentiong what label and value map to within menuItems because the consumer doesn't need to know those implementation details anymore since menuItems is dynamically generated.